src/ui/zcl_abapgit_gui_page_tag.clas.abap

```diff
@@ -157,7 +157,7 @@ CLASS zcl_abapgit_gui_page_tag IMPLEMENTATION.
 
     ro_html->add( render_text_input( iv_name  = 'sha1'
                                      iv_label = 'SHA1'
-                                     iv_value = mo_repo_online->get_sha1_local( ) ) ).
+                                     iv_value = mo_repo_online->get_sha1_remote( ) ) ).
```

Comment:
The SHA1 is open for input in the user interface, but hardcoded to `iv_new         = io_repo->get_sha1_remote( )` in method CREATE_LIGHTWEIGHT_TAG

Reply:
Good catch, fixed.